workflow/automation/lib/MgmtDB.py

```diff
@@ -224,6 +223,72 @@ def update_entries_live(
 
         return True
 
+    def get_core_hour_states(self, rel_name: str, ch_count_type: ChCountType):
+        with connect_db_ctx(self._db_file) as cur:
+            if ch_count_type == ChCountType.Needed:
+                states = []
+                # Selects only tasks after the last failed attempt
+                proc_types = cur.execute(
+                    "SELECT DISTINCT proc_type from state WHERE run_name=? AND status != ?",
+                    (rel_name, const.Status.created.value),
+                ).fetchall()
+                for proc_type in proc_types:
+                    proc_type = proc_type[0]
+                    failed_task_modified = cur.execute(
+                        "SELECT last_modified from state WHERE run_name=? AND status == ? AND proc_type=?",
```

Comment:
For consistency with the rest of the file you should only use a single =. From my brief google, SQL doesn't have a difference between = and ==